src/js/gi/components/RadioButtons.jsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import _ from 'lodash';
+
+import ToolTip from '../../common/components/form-elements/ToolTip';
+import ExpandingGroup from '../../common/components/form-elements/ExpandingGroup';
+
```

Comment:
(The Checkbox and RadioButtons components were copied here to make that possible in a future iteration, but my needs were simpler and significantly different such that I didn't want to tackle generalizing common components right away. (Aside from making it easy to diff to what the diffs are between the common and gibct components.) I thought about using inheritance but that seemed less practical and would've introduced a dependency on the common components that I'm not sure is a good idea.)